src/instructlab/data/generate.py

```diff
@@ -0,0 +1,218 @@
+
+import click
+from instructlab import helpers
+import logging
+
+@click.command()
+@click.option(
+    "--model",
+    default=helpers.DEFAULT_MODEL,
+    show_default=True,
+    help="Name of the model used during generation.",
+)
+@click.option(
+    "--num-cpus",
+    type=click.INT,
+    help="Number of processes to use.",
```

Comment:
Number of processes or number of CPUs? Not necessarily the same thing

Reply:
True, this was from before. But looking at the code this seems to actually be the amount of processes to spawn not the # of cpus